app/assets/stylesheets/pageflow/page_transitions/scroll_over_from_right.scss

```diff
@@ -0,0 +1,12 @@
+section.scroll_over_from_right {
+  &.animate-in-forwards, &.animate-out-backwards {
```

Comment:
Selector `animate-in-forwards` should be written in lowercase with underscores<br>Selector `animate-out-backwards` should be written in lowercase with underscores<br>Each selector in a comma sequence should be on its own single line